pkgs/applications/video/handbrake/default.nix

```diff
@@ -75,7 +75,6 @@ stdenv.mkDerivation rec {
   configureFlags = [
     "--disable-df-fetch"
     "--disable-df-verify"
-    "--enable-fdk-aac"
```

Comment:
Could you make this optional like `useGtk` below, so that users could still enable it easily?

Reply:
I didn't knew there is such possibility.
Ok. I would redo it now.

Reply:
Yes, you can install it using `pkgs.handbrake.override { useFdk = true; }`.

Reply:
Oh. Yes, you are right.

I used those options, but newer implemented them until now.